core/storage/topic/gae_models.py

```diff
@@ -220,6 +240,8 @@ class TopicSummaryModel(base_models.BaseModel):
     language_code = ndb.StringProperty(required=True, indexed=True)
     # The description of the topic.
     description = ndb.TextProperty(indexed=False)
+    # The url fragment of the topic.
+    url_fragment = ndb.StringProperty(indexed=True)
```

Comment:
Do we need a TODO/issue to make this required=True?

Reply:
Done.